scapy/sendrecv.py

```diff
@@ -897,7 +903,7 @@ def _select(sockets):
         read_allowed_exceptions = (PcapTimeoutElapsed,)
 
         def _select(sockets):
-            return sockets
+            return select_objects(sockets, remain=0)
```

Comment:
Note to self: separate
- `WINDOWS and started_callback` ==> use select_objects
- `WINDOWS and not started_callback` ==> previous behavior: dont use select_objects